yktracec/src/jitmodbuilder.cc

```diff
@@ -1714,7 +1711,7 @@ class JITModBuilder {
                   // main interpreter.
                   BasicBlock *FailBB =
                       getGuardFailureBlock(BB, CurBBIdx, CPInstr, CurInstrIdx,
-                                           SIDETRACE_LAST_GUARD_ID);
+                                           GuardCount);
```

Comment:
I hope this won't bite us later. Although I can't currently think of a way that this guard may be created before other guards. It could be an issue when we start patching traces, but then we need to rework the guards anyway, and there probably won't be such a thing as the last guard anymore. This I believe this is fine for now.

Reply:
If we do need that, I reckon it'll be fairly obvious what's gone wrong.